Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -3482,17 +3483,17 @@ def com_google_fonts_check_040(ttFont, vmetrics):
 
 @check(
   id = 'com.google.fonts/check/042',
-  rationale = """When OS/2 and hhea vertical metrics match, the same 
+  rationale = """When OS/2 and hhea vertical metrics match, the same
   linespacing results on macOS, GNU+Linux and Windows. Unfortunately as of 2018,
-  Google Fonts has released many fonts with vertical metrics that don't match 
-  in this way. When we fix this issue in these existing families, we will 
+  Google Fonts has released many fonts with vertical metrics that don't match
+  in this way. When we fix this issue in these existing families, we will
   create a visible change in line/paragraph layout for either Windows or macOS
-  users, which will upset some of them. 
+  users, which will upset some of them.
 
   But we have a duty to fix broken stuff, and inconsistent paragraph layout is
-  unacceptably broken when it is possible to avoid it. 
-  
-  If users complain and prefer the old broken version, they are libre to take 
```

Comment:
thanks for the white-space cleanup in here :-D

Reply:
Oh... that wasn't intentional. I think I have Sublime set up to automatically remove trailing whitespace. I usually work on Font Bakery in PyCharm, but I just used Sublime to find & replace all instances of the name, and accidentally cleaned up the whitespace in the process... and then failed to notice it in my pre-commit diff, sorry. 😊

Reply:
no worries!